tests/publiccloud/run_ltp.pm

```diff
@@ -36,21 +36,34 @@ sub wait_for_guestregister
 
 sub run {
     my ($self) = @_;
-    my $ltp_repo = get_var('LTP_REPO', 'https://download.opensuse.org/repositories/home:/metan/' . generate_version() . '/home:metan.repo');
+    my $ltp_repo = get_var('LTP_REPO', 'http://download.suse.de/ibs/QA:/Head/' . generate_version("-") . '/x86_64/');
+    my $REG_CODE = get_required_var('SCC_REGCODE');
     $self->select_serial_terminal;
 
+    my $out       = script_output("curl $ltp_repo | grep ltp | head -1");
```

Comment:
Only as a hint. The https://mojolicious.org/perldoc/Mojo/UserAgent offers a nice way to retrieve/parse html pages.
For instance we use it in `trigger_public_cloud.pl` script.